.ci.yaml

```diff
@@ -302,11 +302,13 @@ targets:
           {"dependency": "open_jdk", "version": "version:11"}
         ]
 
+  # All of the Linux_android android_platform_tests shards have the same
+  # dependency list, despite some running on Android 33 AVDs versus 34.
   - name: Linux_android android_platform_tests_shard_1 master
     recipe: packages/packages
     timeout: 60
     properties:
-      target_file: android_platform_tests.yaml
+      target_file: android_platform_tests_api_33.yaml
       channel: master
       version_file: flutter_master.version
       package_sharding: "--shardIndex 0 --shardCount 6"
```

Comment:
You'll need to changed the shard flags (the names of the tasks will be slightly confusing during the transition, but changing names is a huge pain with LUCI), so that each of the two partitions has their own shards, otherwise we may end up with a plugin assigned to a specific shard, but then excluded in that shard (since sharding is pre-exclusion, for slightly complicated reasons).

So I believe the setup we need is something like:
```
- name: ..._shard_1
  // set up for 34
  package_sharding: "--shardIndex 0 --shardCount 4"

- name: ..._shard_2
  // set up for 34
  package_sharding: "--shardIndex 1 --shardCount 4"

- name: ..._shard_3
  // set up for 34
  package_sharding: "--shardIndex 2 --shardCount 4"

- name: ..._shard_4
  // set up for 34
  package_sharding: "--shardIndex 3 --shardCount 4"

- name: ..._shard_5
  // set up for 33
  package_sharding: "--shardIndex 0 --shardCount 1"

- name: ..._shard_6
  // set up for 33
  package_sharding: "--shardIndex 1 --shardCount 2"
```

Reply:
The other option we could consider if we think we'll be in this state for a while is to bring up a new task or two for 33 tests, and leave the existing 6 shards as-is but with 34, so the names make more sense.

Reply:
I'm going to be actively working on getting the tests failing on 34 working, but two of the failures have been really tricky, so could be a matter of months worst case. I'll start changing the shard flags but let me know if you think it makes more sense to add new tasks instead given that projection.

Reply:
I think on a potential timeline of months it's probably better to bring up 1-2 new shards for 33.

Reply:
Ok hoping to be faster but I think that's probably safest.